userfunc.go

```diff
@@ -23,8 +23,14 @@ func createUserFuncsObj(contents []string) UserFuncs {
 	}
 
 	userFuncObj := UserFuncs{
-		Time:       getIntFromStr(contents[0]),
-		UserID:     "user_" + contents[1],
+		Time: getIntFromStr(contents[0]),
+		UserID: func() string {
```

Comment:
Appending "user_" to the user id from the input file only when its a number. If its a string, I'll just use the given string from the input file as user id.